src/test/java/neworangehrm/com/opensourcehrm/steps/AddUserSteps.java

```diff
@@ -0,0 +1,26 @@
+package neworangehrm.com.opensourcehrm.steps;
+
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import neworangehrm.com.opensourcehrm.util.DriverUtil;
+import org.openqa.selenium.WebDriver;
+
+
+/**
+ * Created by Viswa on 12/27/2017.
+ */
+public class AddUserSteps {
+    DriverUtil driverUtil = new DriverUtil();
+    WebDriver driver = driverUtil.getDriverInstance();
+
+    @And("^I click on Add button$")
+    public void iClickOnAddButton() throws Throwable {
```

Comment:
If at all if we need to throw an exception it should be specific and match the error/exception.

Reply:
Point noted.
And could not fix it as the class file is been deleted to baseline the code to V1.0. Thanks.